client/client.go

```diff
@@ -968,6 +975,7 @@ func (c *Client) AwaitConverged() error {
 		if done {
 			return nil
 		}
+		time.Sleep(time.Second) // avoid busy looping.
```

Comment:
This might increase the execution times for local clients significantly -- do you want to make this 100msec? I agree with avoiding busy-looping but I'm wondering if once a second is often enough to check.

Reply:
Sounds good! Updated interval to 100 * time.Millisecond.